testsuite/run_sets/secondary.yml

```diff
@@ -8,6 +8,7 @@
 
 # IDEMPOTENT
 
+- features/secondary/srv_manage_activation_keys.feature
```

Comment:
Let's do something, as this feature is completely harmless for other features, let's move it to the other yml file, to "/run_sets/secondary_parallelizable.yml".
In that way, when we activate the parallelization, we can run this feature in parallel with others.